salesforcecli/cli
# Replace `plugin-config` with `plugin-settings`

### What does this PR do?
`plugin-config v2` has been moved to `plugin-settings` ([source](https://github.com/salesforcecli/plugin-settings)) 
This PR swaps out the two, there should be no functional differences

### QA Notes
- Confirm `config` commands still work
- Confirm the `plugin-config` package is removed
- Confirm all references to `plugin-config` are gone

### What issues does this PR fix or reference?
[@W-11762876@](https://gus.my.salesforce.com/apex/ADM_WorkLocator?bugorworknumber=W-11762876)
